src/Dropdown/Dropdown.js

```diff
@@ -1,15 +1,28 @@
-import React, { Fragment } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { css } from 'styled-components';
 import Popper from 'popper.js';
 import Box from '../Box';
 import Portal from '../Portal';
 import { createComponent, themeGet } from '../utils';
 
+const blurEvents = {
+  click: 'blur',
+  mouseenter: 'mouseleave',
+};
+
+const focusEvents = {
+  click: 'focus',
+  mouseenter: 'mouseenter',
+};
+
 const DropdownTrigger = createComponent({
   name: 'DropdownTrigger',
   style: css`
     display: inline-block;
+    &:focus {
+      outline: none;
```

Comment:
This is an accessibility no no: 
> Buttons need a fully visible keyboard focus outline so users can see them when tabbing.
https://www.deque.com/blog/accessible-aria-buttons/

Should we create an accessibility audit task, and include issues like this as sub-tasks?

Reply:
i think thats a good diea - there's probably a lot of accessibility oopsies to be addressed anyways